app/main.py

```diff
@@ -1,3 +1,58 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: Union[int, float]) -> Distance:
```

Comment:
The `__init__` method should not have a return type annotation. Constructors in Python do not return a value, so the `-> Distance` part should be removed.